_P208_Nokia_LCD_5110.ino

```diff
@@ -271,12 +277,19 @@ boolean displayText3(char deviceTemplate[][48], struct EventStruct *event ){ //4
             lcd3->setTextSize(1);
           }
           String tmpString = deviceTemplate[x];
-          String newString = "";
+          String newString = " ";
+          String newString3 = " ";

```

Comment:
You assign something to the `newString` later, so no need to first initialize it with a space.
The default constructor makes an empty string.

Reply:
Changed in next commit